roles/StackStorm.mongodb/tasks/mongodb_dnf.yml

```diff
@@ -0,0 +1,54 @@
+---
+- name: yum | Install mongodb dependencies
```

Comment:
There looks like still `mongodb_dnf.yml` file still present. Probably missed to remove in previous iteration.

Reply:
I left that one in as every single task was different between RH7 and RH8 for mongo. I could keep as a single _yum.yml but then each task will need to import RH7/RH8 specific variables to know what to import. But can go that way if preferred.

Reply:
Yes, having el7/el8 specific variables to import is a widely used practice and would look more clear. Just included an  example in other comment.